_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { closePopupsIfExist } from './helpers/helpers';
+import { deleteAllPopularRepos } from './helpers/deletePopularRepositories';
+
+test.describe('Popular Repositories', () => {
+  test('Add popular repos', async ({ page }) => {
+    // Ensure no popular repos are selected.
+    await deleteAllPopularRepos(page);
+
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+    await expect(page).toHaveTitle('Repositories - Content | RHEL');
+
+    await test.step('Select the Popular repos tab', async () => {
+      await page.getByRole('link', { name: 'Popular repositories' }).click();
+      await expect(page.getByTestId('popular_repos_table')).toBeVisible();
+      await page.getByRole('button', { name: 'Add selected repositories' });
```

Comment:
Is this enough? Don't you need to assert something? E.g. "toBeVisible()` or do something e.g. `click()`?

Reply:
We do not need to click as that is the expected default state, I can change that last line to use "expect"

Reply:
right, when we just need to have something visible, then `toBeVisible()` should be used.